### PR TITLE
BZ2075072-B Adds Note for public & private subnet tags

### PR DESCRIPTION
**Pull request by Wiharris**
**BZ2075072-B**
Adds Note for public & private subnet tags

**Version(s):**
  * PR applies to all versions after a specific version (e.g. 4.6): 4.6+

**Issue:**
https://bugzilla.redhat.com/show_bug.cgi?id=2075072#c6

**Link to docs preview:**

- [Installing a cluster on AWS with network customizations](https://49764--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations.html)
- [CloudFormation template for the network and load balancers](https://wiharris.github.io/openshift-docs/BZ2075072-B/installing/installing_aws/installing-aws-user-infra.html#installation-cloudformation-dns_installing-aws-user-infra)

**Additional information:**
duplicate of closed PR: https://github.com/openshift/openshift-docs/pull/49626

QE: @yunjiang29

### CODE REVIEW

File: installing/installing_aws/installing-aws-network-customizations.adoc [hunk 1]
```diff
@@ -16,6 +16,11 @@ You must set most of the network configuration parameters during installation,
 and you can modify only `kubeProxy` configuration parameters in a running
 cluster.
 
+[IMPORTANT]
+====
+Ensure the IngressController's public and private subnets are correctly tagged for Kubernetes to pick the correct internal load balancer. For more information about the proper tags for public and private subnets, see link:https://docs.openshift.com/container-platform/4.8/installing/installing_aws/installing-aws-user-infra.html#installation-cloudformation-dns_installing-aws-user-infra[CloudFormation template for the network and load balancers] in the AWS documentation.
```
Comment (yunjiang29) [inline installing/installing_aws/installing-aws-network-customizations.adoc:21]: why specify a particular version (4.8) here, I expect the same version as the current doc version. 
Also, this article is generally for IPI installation, but this note points to the UPI doc, which may confuse users.

File: modules/installation-cloudformation-dns.adoc [hunk 2]
```diff
@@ -9,6 +9,16 @@
 You can use the following CloudFormation template to deploy the networking
 objects and load balancers that you need for your {product-title} cluster.
 
+[IMPORTANT]
+====
+If you are deploying the networking objects and load balancers, you must ensure the IngressController's public and private subnets are correctly tagged for Kubernetes to pick the correct internal load balancer. For example:
+[source,yaml]
+----
+RegisterPublicSubnetTags: http://kubernetes.io/role/elb
+RegisterPrivateSubnetTags: http://kubernetes.io/role/internal-elb
```
Comment (yunjiang29) [inline modules/installation-cloudformation-dns.adoc:18]: 1. this format is not good for users,  they may think the `RegisterPublicSubnetTags` is the key, and `http://kubernetes.io/role/internal-elb` is the value, all things present as yaml format would be considered  as _code_ (tag key/value), not  description (_RegisterPublicSubnetTags_)  
2. the key is `kubernetes.io/role/internal-elb`, value is _[empty]_

**Comment by yunjiang29** [comment]
@lihongan PTAL, thanks.

File: modules/installation-cloudformation-dns.adoc (hunk 2 above)
Comment (lihongan) [reply modules/installation-cloudformation-dns.adoc:18]: Tes, the tag format is a key-value pair, here the key should be `kubernetes.io/role/internal-elb` and no `http://`.

File: installing/installing_aws/installing-aws-network-customizations.adoc (hunk 1 above)
Comment (Wiharris) [reply installing/installing_aws/installing-aws-network-customizations.adoc]: Done. :)

**Comment by Wiharris** [comment]
@yunjiang29 I have updated the PR. Please review.

**Comment by yunjiang29** [comment]
@lihongan can you help review this PR?

File: modules/installation-cloudformation-dns.adoc [hunk 3]
```diff
@@ -9,6 +9,16 @@
 You can use the following CloudFormation template to deploy the networking
 objects and load balancers that you need for your {product-title} cluster.
 
+[IMPORTANT]
+====
+If you are deploying the networking objects and load balancers, you must ensure the IngressController's public and private subnets are correctly tagged for Kubernetes to pick the correct internal load balancer. For example:
+[source,yaml]
+----
+RegisterPublicSubnetTags: http://kubernetes.io/role/elb
+RegisterPrivateSubnetTags: kubernetes.io/role/internal-elb
```
Comment (yunjiang29) [inline modules/installation-cloudformation-dns.adoc:18]: I prefer something like:

Register public subnet tags:
```
kubernetes.io/cluster/<infra-id>: shared or owned
```
Register private subnet tags:
```
kubernetes.io/cluster/<infra-id>: shared or owned
kubernetes.io/role/internal-elb: <empty>
```

@lihongan WDYT? refer to https://issues.redhat.com/browse/OCPQE-10362?focusedCommentId=20332409&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20332409

**Pull request closed without merging**